inc/class-wpseo-meta.php

```diff
@@ -1002,5 +1007,25 @@ public static function get_post_value( $key ) {
 		return ( array_key_exists( $key, $_POST ) ) ? $_POST[ $key ] : '';
 	}
 
+	/**
+	 * Counts the total of all the keywords being used for posts except the given one
+	 *
+	 * @param string  $keyword
+	 * @param integer $post_id
+	 *
+	 * @return array
+	 */
+	public static function keyword_usage( $keyword, $post_id ) {
+		return get_posts(
```

Comment:
We should always try to use `WP_Query` instead of `get_posts`. Because `get_posts` is just a wrapper around `WP_Query`. By default `get_posts` has `suppress_filters` on.

You can use the following for example:
```php
$posts = new WP_Query( $args );
return $posts->posts;
```